.spelling

```diff
@@ -1,5 +1,498 @@
-Kubernetes
+100m
```

Comment:
It seems like quite a few here are from code snippets? Our spellchecker should skip text in backticks, which I think should make a lot of these not necessary?